packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts

```diff
@@ -48,6 +48,14 @@ withChainsDescribe("Tokens reducer", () => {
           tokenTicker: "ASH",
         },
       },
+      ASHETH: {
```

Comment:
Now that #863 is merged, can you please rebase this to master and rename this token to the original name `ASHETH` -> `ASH`? Thanks